platform/security/cas/security-cas-client/src/test/java/org/codice/ddf/security/handler/cas/CasHandlerTest.java

```diff
@@ -147,7 +147,7 @@ public void testCachedPrincipalResolve() throws ServletException, IOException {
 
     private CasHandler createHandler() {
         CasHandler handler = new CasHandler();
-        STSClientConfiguration clientConfiguration = new STSClientConfigurationImpl();
+        STSClientConfiguration clientConfiguration = new StsRealm();
```

Comment:
I like that idea a lot better, it should never have had that test dependency in the first place.